webapp/src/App.tsx

```diff
@@ -18,6 +19,7 @@ function App() {
           {/* Using the same element since it's handling both. Normally you'd probably have a separate one */}
           <Route path=":displayName" element={<CreatedBy />} />
         </Route>
+        <Route path="/newbie/:displayName" element={<ButtonArsh />} />
```

Comment:
Awesome!

Also FYI the `:displayName` part represents a variable, since you're not making use of that you _could_ (but don't have to!) make the path just `/newbie` so http://localhost:3000/newbie would take you to the page.